src/user-event/utils/text-range.ts

```diff
@@ -0,0 +1,11 @@
+export interface TextRange {
+  start: number;
+  end: number;
+}
+
+export function getTextRange(text: string): TextRange {
+  return {
+    start: text.length,
```

Comment:
Why do start and end have the same value?

Reply:
This is how the event is passed on regular typing, it means that the cursor is at that position, but no text is actually seleted.